src/main/cluster.ts

```diff
@@ -433,6 +435,18 @@ export class Cluster implements ClusterModel, ClusterState {
     this.metadata = Object.assign(existingMetadata, metadata);
   }
 
+  /**
+   * @internal
+   */
+  async refreshAccessibility(): Promise<void> {
```

Comment:
This probably could be `private` because it should not be called from outside.

Reply:
Agreed

Reply:
Fixed in #2063